packages/playground/app/api/notify/route.ts

```diff
@@ -0,0 +1,33 @@
+import { sendFrameNotification } from "@/lib/notification-client";
+
+export async function POST(request: Request) {
+  try {
+    const body = await request.json();
+    const { fid, notification } = body;
+
+    const result = await sendFrameNotification({
+      fid,
+      title: notification.title,
+      body: notification.body,
+    });
+
+    if (result.state === "error") {
+      return new Response(JSON.stringify({ error: result.error }), {
+        status: 500,
+        headers: { "Content-Type": "application/json" },
+      });
```

Comment:
suggestion(non-blocking): Just FYI, there's a `NextResponse.json()` util that lets you skip stringifying and setting content type.

E.g.

```ts
NextResponse.json({ error: result.error }, { status: 500 })
```